nextflow.config

```
@@ -214,7 +214,7 @@ manifest {
     description     = 'Identify and quantify peptides from mass spectrometry raw data'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=21.10.3'
-    version         = '2.1.0'
+    version         = '2.1.1dev'
```

You are merging from `dev` to `master`, so shouldnt be here
```suggestion
    version         = '2.2'
```
or 
`    version         = '2.2.0'`?